File: src/java/src/quickfix/fix43/ListExecute.java
package quickfix.fix43;
import quickfix.FieldNotFound;
import quickfix.Group;
import quickfix.field.*;

public class ListExecute extends Message
{
  public ListExecute()
  {
    getHeader().setField(new MsgType("L"));
  }
  public ListExecute(
    quickfix.field.ListID aListID,
    quickfix.field.TransactTime aTransactTime ) {

    getHeader().setField(new MsgType("L"));
    set(aListID);
    set(aTransactTime);
  }

  public void set(quickfix.field.ListID value)
  { setField(value); }
  public quickfix.field.ListID get(quickfix.field.ListID  value)
    throws FieldNotFound
  { getField(value); return value; }
  public quickfix.field.ListID getListID() throws FieldNotFound
  { quickfix.field.ListID value = new quickfix.field.ListID();
    getField(value); return value; }
  public boolean isSet(quickfix.field.ListID field)
  { return isSetField(field); }
  public boolean isSetListID()
  { return isSetField(66); }
  public void set(quickfix.field.ClientBidID value)
  { setField(value); }
  public quickfix.field.ClientBidID get(quickfix.field.ClientBidID  value)
    throws FieldNotFound
  { getField(value); return value; }
  public quickfix.field.ClientBidID getClientBidID() throws FieldNotFound
  { quickfix.field.ClientBidID value = new quickfix.field.ClientBidID();
    getField(value); return value; }
  public boolean isSet(quickfix.field.ClientBidID field)
  { return isSetField(field); }
  public boolean isSetClientBidID()
  { return isSetField(391); }
  public void set(quickfix.field.BidID value)
  { setField(value); }
  public quickfix.field.BidID get(quickfix.field.BidID  value)
    throws FieldNotFound
  { getField(value); return value; }
  public quickfix.field.BidID getBidID() throws FieldNotFound
  { quickfix.field.BidID value = new quickfix.field.BidID();
    getField(value); return value; }
  public boolean isSet(quickfix.field.BidID field)
  { return isSetField(field); }
  public boolean isSetBidID()
  { return isSetField(390); }
  public void set(quickfix.field.TransactTime value)
  { setField(value); }
  public quickfix.field.TransactTime get(quickfix.field.TransactTime  value)
    throws FieldNotFound
  { getField(value); return value; }
  public quickfix.field.TransactTime getTransactTime() throws FieldNotFound
  { quickfix.field.TransactTime value = new quickfix.field.TransactTime();
    getField(value); return value; }
  public boolean isSet(quickfix.field.TransactTime field)
  { return isSetField(field); }
  public boolean isSetTransactTime()
  { return isSetField(60); }
  public void set(quickfix.field.Text value)
  { setField(value); }
  public quickfix.field.Text get(quickfix.field.Text  value)
    throws FieldNotFound
  { getField(value); return value; }
  public quickfix.field.Text getText() throws FieldNotFound
  { quickfix.field.Text value = new quickfix.field.Text();
    getField(value); return value; }
  public boolean isSet(quickfix.field.Text field)
  { return isSetField(field); }
  public boolean isSetText()
  { return isSetField(58); }
  public void set(quickfix.field.EncodedTextLen value)
  { setField(value); }
  public quickfix.field.EncodedTextLen get(quickfix.field.EncodedTextLen  value)
    throws FieldNotFound
  { getField(value); return value; }
  public quickfix.field.EncodedTextLen getEncodedTextLen() throws FieldNotFound
  { quickfix.field.EncodedTextLen value = new quickfix.field.EncodedTextLen();
    getField(value); return value; }
  public boolean isSet(quickfix.field.EncodedTextLen field)
  { return isSetField(field); }
  public boolean isSetEncodedTextLen()
  { return isSetField(354); }
  public void set(quickfix.field.EncodedText value)
  { setField(value); }
  public quickfix.field.EncodedText get(quickfix.field.EncodedText  value)
    throws FieldNotFound
  { getField(value); return value; }
  public quickfix.field.EncodedText getEncodedText() throws FieldNotFound
  { quickfix.field.EncodedText value = new quickfix.field.EncodedText();
    getField(value); return value; }
  public boolean isSet(quickfix.field.EncodedText field)
  { return isSetField(field); }
  public boolean isSetEncodedText()
  { return isSetField(355); }
}

File: src/.NET/fix42/ListExecute.cs
namespace QuickFix42
{

  public class ListExecute : Message
  {
    public ListExecute() : base(MsgType()) {}
    static QuickFix.MsgType MsgType() { return new QuickFix.MsgType("L"); }

    public ListExecute(
      QuickFix.ListID aListID,
      QuickFix.TransactTime aTransactTime )
    : base(MsgType()) {
      set(aListID);
      set(aTransactTime);
    }

    public void set(QuickFix.ListID value)
    { setField(value); }
    public QuickFix.ListID get(QuickFix.ListID  value)
    { getField(value); return value; }
    public QuickFix.ListID getListID()
    { QuickFix.ListID value = new QuickFix.ListID();
      getField(value); return value; }
    public bool isSet(QuickFix.ListID field)
    { return isSetField(field); }
    public bool isSetListID()
    { return isSetField(66); }

    public void set(QuickFix.ClientBidID value)
    { setField(value); }
    public QuickFix.ClientBidID get(QuickFix.ClientBidID  value)
    { getField(value); return value; }
    public QuickFix.ClientBidID getClientBidID()
    { QuickFix.ClientBidID value = new QuickFix.ClientBidID();
      getField(value); return value; }
    public bool isSet(QuickFix.ClientBidID field)
    { return isSetField(field); }
    public bool isSetClientBidID()
    { return isSetField(391); }

    public void set(QuickFix.BidID value)
    { setField(value); }
    public QuickFix.BidID get(QuickFix.BidID  value)
    { getField(value); return value; }
    public QuickFix.BidID getBidID()
    { QuickFix.BidID value = new QuickFix.BidID();
      getField(value); return value; }
    public bool isSet(QuickFix.BidID field)
    { return isSetField(field); }
    public bool isSetBidID()
    { return isSetField(390); }

    public void set(QuickFix.TransactTime value)
    { setField(value); }
    public QuickFix.TransactTime get(QuickFix.TransactTime  value)
    { getField(value); return value; }
    public QuickFix.TransactTime getTransactTime()
    { QuickFix.TransactTime value = new QuickFix.TransactTime();
      getField(value); return value; }
    public bool isSet(QuickFix.TransactTime field)
    { return isSetField(field); }
    public bool isSetTransactTime()
    { return isSetField(60); }

    public void set(QuickFix.Text value)
    { setField(value); }
    public QuickFix.Text get(QuickFix.Text  value)
    { getField(value); return value; }
    public QuickFix.Text getText()
    { QuickFix.Text value = new QuickFix.Text();
      getField(value); return value; }
    public bool isSet(QuickFix.Text field)
    { return isSetField(field); }
    public bool isSetText()
    { return isSetField(58); }

    public void set(QuickFix.EncodedTextLen value)
    { setField(value); }
    public QuickFix.EncodedTextLen get(QuickFix.EncodedTextLen  value)
    { getField(value); return value; }
    public QuickFix.EncodedTextLen getEncodedTextLen()
    { QuickFix.EncodedTextLen value = new QuickFix.EncodedTextLen();
      getField(value); return value; }
    public bool isSet(QuickFix.EncodedTextLen field)
    { return isSetField(field); }
    public bool isSetEncodedTextLen()
    { return isSetField(354); }

    public void set(QuickFix.EncodedText value)
    { setField(value); }
    public QuickFix.EncodedText get(QuickFix.EncodedText  value)
    { getField(value); return value; }
    public QuickFix.EncodedText getEncodedText()
    { QuickFix.EncodedText value = new QuickFix.EncodedText();
      getField(value); return value; }
    public bool isSet(QuickFix.EncodedText field)
    { return isSetField(field); }
    public bool isSetEncodedText()
    { return isSetField(355); }

  };

}

